### ISSUE TITLE
Hostname pattern support for ignoring domains

### ISSUE THREAD

**Issue by n370** [JannesMeyer/TabAttack#15]
Much like the way you would do to manage cookie and site data exceptions.

[Manage exceptions](https://support.google.com/chrome/answer/3123708?p=settings_manage_exceptions&rd=1)

[Match patterns](https://developer.chrome.com/extensions/match_patterns)

**Comment by JannesMeyer** [comment]
Yes, I totally agree that that makes sense. Do you know of any library for that kind of chrome-style url pattern matching? I feel like this matching could be useful to a lot of other projects and should be a library. If it doesn't exist yet then I'm gonna consider making it myself as it doesn't sound that complicated :)

**Comment by n370** [comment]
Hey @JannesMeyer! No, I'm not aware of anything like that but will certainly come back here as soon as I find something. Anyway I would probably look for anything RegExp or UNIX glob related if could find some time to dive deep into write the code myself. Probably the Chrome/Chromium source code and it's project discussion lists are also good places to look for. I'm looking forward to contribute code myself to this project, unfortunately by now I'm just able to open issues, I hope it doesn't bothers you.
